certification/engine/engine.go

```diff
@@ -89,8 +112,8 @@ var (
 	hasNoProhibitedCheck   certification.Check = &containerpol.HasNoProhibitedPackagesCheck{}
 	hasRequiredLabelsCheck certification.Check = &containerpol.HasRequiredLabelsCheck{}
 	runAsRootCheck         certification.Check = &containerpol.RunAsNonRootCheck{}
-	basedOnUbiCheck        certification.Check = &containerpol.BasedOnUBICheck{}
 	hasModifiedFilesCheck  certification.Check = &containerpol.HasModifiedFilesCheck{}
+	basedOnUbiCheck        certification.Check = containerpol.NewBasedOnUbiCheck(&PyxisLayerHashChecker{})
```

Comment:
This should still be passing in PyxisEngine, as it would satisfy the HashChecker interface.

Reply:
yeah anything that satisfies the interface 👍